pkg/rkenodeconfigserver/nodeserver_test.go

```diff
@@ -62,6 +62,14 @@ func TestAppendKubeletArgs(t *testing.T) {
 	}
 }
 
+func TestShareMntArgs(t *testing.T) {
+	augmentedProcesses := getAugmentedKubeletProcesses()
+	args := augmentedProcesses["share-mnt"].Args
```

Comment:
do we need to add check for this key in map?

Reply:
We pass b2d bool true in test, that ensures `share-mnt` will be present in processes map

Reply:
worker=true, b2d=false in our test but its based on wether there are binds with `shared`, if so, `share-mnt` gets added.